h/templates/layouts/account.html.jinja2

```diff
@@ -17,7 +17,6 @@
   {{ panel('navbar') }}
   {% endif %}
   <div class="content paper">
-    {% include "h:templates/includes/logo-header.html.jinja2" %}
```

Comment:
Doesn't need to be fixed in this PR but I'll note that the tabs now touch the navbar and need some vertical spacing above them. For the logged-out pages this is provided by the `form-header` class.